hbw/app/models/hbw/fields/select.rb

```diff
@@ -59,6 +60,10 @@ def choices_sql
         sql
       end
 
+      def placeholder
+        definition.fetch('placeholder')
```

Comment:
What would happen if no "placeholder" specified in config? "Placeholder" is not a required option.